### ISSUE TITLE
ERL-77: common test code coverage does not honour excl_dirs etc to exclude directories

### ISSUE THREAD

**Issue by OTP-Maintainer** [erlang/otp#3386]
Original reporter: `linearregression`
Affected version: `OTP-18.0`
Component: `common_test`
Migrated from: https://bugs.erlang.org/browse/ERL-77

---

```
Per http://www.erlang.org/doc/apps/common_test/cover_chapter.html
We should eb able to exclude directories, modules etc.
{code}%% Directories to exclude in cover.
{excl_dirs, Dirs},
{excl_dirs_r, Dirs}.{code}
But it seems like ot being honored notice in 18.2,18.1, 18.0.Have not tried other versions.

Output after runnjng code coverage below shows the list of test code to exclude:
content of coverpec
{code}{nodes, ['gen_rpc_master@127.0.0.1', 'gen_rpc_slave@127.0.0.1', 'nonode@nohost']}.
{export, "../_build/test/cover/ct.coverdata"}.
{level, details}.
{incl_dirs_r, ["../src"]}.
{excl_dirs_r, ["../test"]}.{code}
using excl_mods does not work either.
Basically all the exclude doe snot seem to work.
This is basic functionality of a test tool. 
Can some one assigned to fix?

Btu end results
{code}|----------------------------------|------------|
| module | coverage |
|----------------------------------|------------|
| local_functional_SUITE | 98% |
| multi_rpc_functional_SUITE | 87% |
| gen_rpc_test_helper | 79% |
| remote_functional_SUITE | 96%{code}

Originally reported here https://github.com/priestjim/gen_rpc/issues/42
```
